src/Constructorio_NET.Tests/client/modules/BrowseTests.cs

```diff
@@ -1,9 +1,9 @@
-using System.Collections.Generic;
-using System.IO;
-using System.Threading.Tasks;
 using Constructorio_NET.Models;
 using Newtonsoft.Json.Linq;
 using NUnit.Framework;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
```

Comment:
Is there a reason these were moved here? My lint is telling me to move them back

Reply:
My linter moved them 😂